File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepSendMessageToGroups.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdGroup;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntAutomationStepSendMessageToGroups
  extends StudioEntAutomationStepSendMessageWithSenderField
{
  @Nullable
  public MetaIdGroup[] groupIdSet;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepSendMessageToGroups.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdGroup

open class StudioEntAutomationStepSendMessageToGroups :
    StudioEntAutomationStepSendMessageWithSenderField() {
    var groupIdSet: Array<MetaIdGroup>? = null
}
